test/image/mocks/bar_attrs_group_norm.json

```diff
@@ -1,19 +1,24 @@
 {
     "data":[
         {
-            "base":4,
-            "x":[3,2,1,0],
+            "text":0.5,
+            "textposition":"outside",
+            "outsidetextfont": {"size":32},
+            "base":[100,40,25,10],
+            "x":[-50,10,25,40],
             "type":"bar"
         }, {
-            "base":[7,6,5,4],
-            "x":[1,2,3,4],
+            "text":0.5,
```

Comment:
Adding `textrotation` isn't a bad idea at all. I'm sure users will want to rotate their labels to 45 degrees.